modules/osdk-ansible-k8s-module-installing.adoc

```diff
@@ -9,7 +9,7 @@ To install the `k8s` Ansible module on your local workstation:
 
 .Procedure
 
-. Install Ansible 2.6+:
+. Install Ansible 2.9+:
 +
 ----
 $ sudo yum install ansible
```

Comment:
@sheriff-rh  I'd suggest that replace the `sudo yum install ansible` with the `sudo pip install ansible==2.9`.
One more problem, we also need to install the `kubernetes` module for the ansible. As follows:
```
sudo pip install kubernetes
sudo pip install openshift
```

Reply:
I included `sudo pip install kubernetes`, however i left `sudo yum install ansible` as yum/dnf install the latest available by default. Latest on RHEL 8 and Fedora  is ansible-2.9.10-1.